src/page/blog/article-page.jsx

```diff
@@ -22,7 +22,7 @@ export const Article = ({ type }) => {
   const [successful, setSuccessful] = useState(false);
   const [createPost, { loading, error }] = useMutation(CREATE_POST);
   const { currentUser } = useCurrentUser();
-  const { data } = useQuery(GET_BLOG_POST, {
+  const { data: postData } = useQuery(GET_BLOG_POST, {
```

Comment:
just updated to 'postData' instead of 'data' as there are a lot of queries in this and it made the code easier to read.